domain/src/main/java/com/teamagam/gimelgimel/domain/messages/repository/NetworkStateRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.teamagam.gimelgimel.domain.messages.repository;
+
+public interface NetworkStateRepository {
```

Comment:
why, what the point? I already have class the supply me an observable for network state changes the repository only exist to hold a Boolean field which represent the network state at the time that he being called